me.njean.shootingpactris.appdata.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>me.njean.shootingpactris</id>
+
+  <!--
+      The tag 'metadata_license' means the licence of this file, not the whole
+      product; see https://people.freedesktop.org/~hughsient/appdata/
+      as of 20.09.2013. The CC0 licence is GPL compatible, see
+      https://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses
+      as of 20.09.2013
+  -->
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-3.0+</project_license>
+  <developer_name>Nicolas Jean</developer_name>
+  <name>Shooting Pactris</name>
+  <summary>A coop retro game with Tetris, Pacman and shmup gameplay.</summary>
+  <description>
+    <p>
+      A fun cooperation multiplayer game which mixes universes of
+      Pacman, Tetris and shoot’em up (shmup) games.
+    </p>
+    <p>
+      The player(s) move around Pacman and the ship. They must
+      cooperate to control falling tetris pieces which shoot at the ship,
+      while ghosts follow Pacman in a maze!
+    </p>
+    <p>
+      It can be played on a same machine with the keyboard or with game pads.
+      A network option is also made available.
+    </p>
+  </description>
+  <url type="bugtracker">https://framagit.org/nicooo/shpactris/-/issues</url>
+  <url type="help">https://shootingpactris.njean.me/</url>
+  <url type="homepage">https://shootingpactris.njean.me/</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://shootingpactris.njean.me/img/shpactris1.0.jpg</image>
+    </screenshot>
+  </screenshots>
+  <keywords>
+    <keyword>retrogaming</keyword>
+    <keyword>tetris</keyword>
+    <keyword>pacman</keyword>
+    <keyword>shmup</keyword>
+    <keyword>shoot'em up</keyword>
+    <keyword>multiplayer</keyword>
+  </keywords>
+
+  <launchable type="desktop-id">me.njean.shootingpactris.desktop</launchable>
+  <provides>
+    <binary>shpactris</binary>
+  </provides>
+
+  <releases>
+    <release version="1.0-online-beta3" date="2020-05-21">
+      <url type="details">https://framagit.org/nicooo/shpactris/-/tags/v1.0-online-beta3</url>
+      <description>
+        <_p>
+        Players can now chose a nickname when playing online.
+        </_p>
```

Comment:
```suggestion
        <p>
        Players can now chose a nickname when playing online.
        </p>
```
This file isn't translated by intltools ?

Reply:
Oups sorry about this. Bad copy-paste from a translated appdata file. Obviously this basic first version is not translated.